gpm_api/tests/test_dataset/test_granule_files.py

```diff
@@ -76,9 +77,15 @@ def test_open_granule_on_real_files(tmp_path):
                 ds = open_granule(cut_filepath, scan_mode=scan_mode).compute()
                 ds_expected = xr.open_dataset(processed_filepath).compute()
 
-                # Remove history attribute
-                _ = ds.attrs.pop("history", None)
-                _ = ds_expected.attrs.pop("history", None)
+                for _ds in [ds, ds_expected]:
+                    # Remove history attribute
+                    _ds.attrs.pop("history")
+
+                    # Remove attributes conflicting between python versions
```

Comment:
What was happening between different python versions?

Reply:
There is a string which is either `"Unknown based on WGS 84 ellipsoid"` or `"Unknown based on WGS84 ellipsoid"`, probably because of a different dependency version